k8-util/helm/fluvio-app/templates/sc-deployment.yaml

```diff
@@ -57,7 +57,10 @@ spec:
             - name: authorization-config
               mountPath: /etc/fluvio/authorization
             {{ end }}
+        {{ end }}
+        {{- toYaml .Values.scPod.extraContainers | nindent 8 }}
       volumes:
+      {{ if .Values.tls }}
```

Comment:
Shouldn't this be before `volumes:`?

Reply:
works either way

Reply:
Do you know why the CI failed with:

```
        stderr: Error: INSTALLATION FAILED: YAML parse error on fluvio-app/templates/sc-deployment.yaml: error converting YAML to JSON: yaml: line 35: did not find expected '-' indicator
```

Reply:
maybe it doesn't like the `extraContainers: []` placeholder. it worked before when I had an actual container there. trying to fix now

Reply:
fixed